server/app.js

```diff
@@ -1,14 +1,36 @@
-import express from "express";
-import dotenv from "dotenv";
-import connectDB from "./config/db.js";
-import { join } from "path";
-import morgan from "morgan";
+const express = require("express");
+const dotenv = require("dotenv");
+const connectDB = require("./config/db");
+const { notFound, errorHandler } = require("./middleware/errorMiddleware");
+const path = require("path");
+const cors = require("cors")
+
+const userRoutes = require("./routes/userRoutes")
 
 dotenv.config();
 
 connectDB();
 
 const app = express();
 
+// const __dirname = path.resolve();
+// app.use(express.static(path.join(__dirname, "public")))
+
+app.use(cors());
```

Comment:
Should not need to use cors if you use the proxy like I described in a message above